README.md

```diff
@@ -57,54 +57,54 @@ You can also tell TurboGraft to refresh the page, but exclude certain elements f
 
 It requires your `<form>`, `<a>`, or `<button>` to be marked up with:
 
-* `tg-remote`: (optionally valueless for `<form>`, but requires an HTTP method for links) the HTTP method you wish to call on your endpoint
+* `data-tg-remote`: (optionally valueless for `<form>`, but requires an HTTP method for links) the HTTP method you wish to call on your endpoint
 * `href`: (if node is `<a>` or `<button>`) the URL of the endpoint you wish to hit
-* `refresh-on-success`: (optional) The refresh keys to be refreshed, using the body of the response. This is space-delimited
-* `full-refresh-on-success-except`: (optional) Replaces body except for specififed refresh keys, using the body of the XHR which has succeeded
-* `refresh-on-error`: (optional) The refresh keys to be refreshed, but using body of XHR which has failed. Only works with error 422. If the XHR returns and error and you do not supply a refresh-on-error, nothing is changed
-* `full-refresh-on-error-except`: (optional) Replaces body except for specified refresh keys, using the body of the XHR which has failed.  Only works with error 422
-* `remote-once`: (optional) The action will only be performed once. Removes `remote-method` and `remote-once` from element after consumption
-* `full-refresh`: Rather than using the content of the XHR response for partial page replacement, a full page refresh is performed. If `refresh-on-success` is defined, the page will be reloaded on these keys. If `refresh-on-success` is not defined, a full page refresh is performed. Defaults to true if neither refresh-on-success nor refresh-on-error are provided
-* `tg-remote-norefresh`: Prevents `Page.refresh()` from being called, allowing methods to be executed without updating client state
+* `data-refresh-on-success`: (optional) The refresh keys to be refreshed, using the body of the response. This is space-delimited
+* `data-full-refresh-on-success-except`: (optional) Replaces body except for specififed refresh keys, using the body of the XHR which has succeeded
+* `data-refresh-on-error`: (optional) The refresh keys to be refreshed, but using body of XHR which has failed. Only works with error 422. If the XHR returns and error and you do not supply a refresh-on-error, nothing is changed
+* `data-full-refresh-on-error-except`: (optional) Replaces body except for specified refresh keys, using the body of the XHR which has failed.  Only works with error 422
+* `data-remote-once`: (optional) The action will only be performed once. Removes `data-remote-method` and `data-remote-once` from element after consumption
+* `data-full-refresh`: Rather than using the content of the XHR response for partial page replacement, a full page refresh is performed. If `data-refresh-on-success` is defined, the page will be reloaded on these keys. If `data-refresh-on-success` is not defined, a full page refresh is performed. Defaults to true if neither refresh-on-success nor refresh-on-error are provided
```

Comment:
It's unfortuante that, for whatever reason when I implemented this, I didn't scope them all `tg-*`... :cry: